File: app/src/LightningPlus/java/acr/browser/lightning/activity/MyBrowserActivity.java
package acr.browser.lightning.activity;

import android.os.Bundle;

import com.umeng.analytics.MobclickAgent;

/**
 * Created by jianmin on 17-2-6.
 */

public class MyBrowserActivity extends ThemableBrowserActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    protected void onResume() {
        super.onResume();

        // UMENG
        MobclickAgent.onResume(this);
    }

    @Override
    protected void onPause() {
        super.onResume();

        // UMENG
        MobclickAgent.onPause(this);
    }
}

File: app/src/LightningPlus/java/acr/browser/lightning/base/MyBrowserActivity.kt
package acr.browser.lightning.base

import acr.browser.lightning.browser.activity.BrowserActivity
import android.os.Bundle

abstract class MyBrowserActivity : BrowserActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onResume() {
        super.onResume()
    }

    override fun onPause() {
        super.onPause()
    }

    override fun onDestroy() {
        super.onDestroy()
    }
}